problems/test/problem1_test.rb

```diff
@@ -0,0 +1,26 @@
+require "minitest/autorun"
+require_relative "../lib/problem1"
+
+class Problem1Test < MiniTest::Unit::TestCase
+  def setup
+    @problem1 = Problem1.new

```

Comment:
Tests should give you an insight on how the code will be used. 
Would you instantiate a class, to use a method in it just to get the sum of even numbers in a given array?